js/client-battle.js

```diff
@@ -396,14 +396,15 @@
 						if (move.id === 'Struggle' || move.id === 'Recharge') pp = '&ndash;';
 						if (move.id === 'Recharge') move.type = '&ndash;';
 						if (name.substr(0, 12) === 'Hidden Power') name = 'Hidden Power';
+						var moveType = this.getMoveType(move, switchables[pos]);
```

Comment:
We can make a new object that only has move information in a single function (and it will be usable in the future if there's more changes on moves... like pp or whatever) or use two functions, I think both options are good.